docs/en/operations/settings/settings.md

```diff
@@ -1446,4 +1446,26 @@ Possible values:
 
 Default value: 0.
 
+## min_insert_block_size_rows_for_materialized_views {#min-insert-block-size-rows-for-materialized-views}
+
+Sets minimum number of rows in block which can be inserted into a table by an `INSERT` query. This setting is similar to [min_insert_block_size_rows](#min-insert-block-size-rows), but applied only for blocks inserted into [materialized view](../../sql-reference/statements/create.md#create-view). By adjusting this setting, you control blocks squashing while pushing to materialized view and avoid excessive memory usage.
```

Comment:
If this setting works similarly to another setting, you can just copy the description of that setting (if it is not big) and add specifics for this setting. Don't make a user jump around the document without serious reasons, please :-)
Link to a similar setting you can place in See Also.